### PR TITLE
[css-ruby-1] Bikeshed improvements related to property values

### PR DESCRIPTION
**Pull request by SebastianZ**

### CODE REVIEW

File: css-ruby/Overview.bs [hunk 1]
```diff
@@ -160,14 +160,10 @@ Ruby-specific 'display' Values</h3>
...
 
-	<table class="propdef">
-		<tr>
-			<th>Name:
-			<td>display
-		<tr>
-			<th><a href="#values">New Values</a>:
-			<td>ruby | ruby-base | ruby-text | ruby-base-container | ruby-text-container
-	</table>
+	<pre class="propdef">
```
Comment (tabatkins) [inline css-ruby/Overview.bs]: Put a "partial" class on this as well, so it doesn't try to claim to be defining the 'display' property.

File: css-ruby/Overview.bs [hunk 2]
```diff
@@ -897,38 +893,13 @@ Ruby Formatting Properties</h2>
...
-		<tr>
-			<th>Animatable:
-			<td>no
-		<tr>
-			<th>Canonical order:
-			<td><abbr title="follows order of property value definition">per grammar</abbr>
-	</table>
+	<pre class="propdef">
+	Name: ruby-position
+	Value: [ over | under | inter-character ]
```
Comment (tabatkins) [inline css-ruby/Overview.bs]: Remove the brackets.

File: css-ruby/Overview.bs [hunk 3]
```diff
@@ -1096,32 +1042,14 @@ Sharing Annotation Space: the 'ruby-merge' property</h3>
...
-			<th>Computed value:
-			<td>specified value (except for initial and inherit)
-	</table>
+	<pre class="propdef">
+	Name: ruby-align
+	Value: start | center | space-between | space-around
+	Initial: space-around
+	Applies to: ruby bases, ruby annotations, ruby base containers, ruby annotation containers
+	Inherited: yes
+	Computed value: specified value (except for ''initial'' and ''inherit'')
```
Comment (tabatkins) [inline css-ruby/Overview.bs]: Remove the parenthetical.

**Comment by tabatkins** [comment]
:+1: with suggested changes

**Comment by SebastianZ** [comment]
Thank you for the review!
I've addressed everything in https://github.com/w3c/csswg-drafts/pull/224/commits/9c14f09ca467814aa97931f9ff03def5d9d404cb.

Sebastian

**Pull request merged**